test/Garnet.test/RespSetTest.cs

```diff
@@ -388,6 +388,19 @@ public void CanDoSPOPWithCountCommandLC()
             Assert.AreEqual(expectedResponse, strResponse);
         }
 
+        [Test]
+        public void CanRemoveAllElementsLC()
```

Comment:
I would add this case to the existing CanDoSPOPWithCountCommandLC test

Reply:
Thank you for your review, I think I understand what you mean and have fixed it